composer.json

```diff
@@ -11,6 +11,15 @@
     "require": {
         "marco-c/wp_serve_file": "dev-master"
     },
+    "require-dev": {
+        "stevegrunwell/wp-enforcer": "^0.4.1"
+    },
+    "post-install-cmd": [
+    	"wp-enforcer"
```

Comment:
Should this work even if I don't have `wp-enforcer` globally installed, @bhubbard ?